css/properties/grid-gap.json

```diff
@@ -152,6 +152,54 @@
               "deprecated": false
             }
           }
+        },
+        "calc_values": {
+          "__compat": {
+            "description": "<code>calc()</code> values",
+            "support": {
+              "webview_android": {
+                "version_added": false
+              },
+              "chrome": {
+                "version_added": "68"
+              },
+              "chrome_android": {
+                "version_added": "68"
```

Comment:
`chrome` and `chrome_android` should be `false`. Chrome 68 won't even be in canary for another six weeks yet. Even then, there's no guarantee this will stay in 68 all the way to stable.

By the way, where did you get that information?